packages/@tailwindcss-upgrade/src/template/codemods/theme-to-var.test.ts

```diff
@@ -83,6 +83,10 @@ test.each([
   // that this doesn't end up as the modifier in the candidate itself.
   ['max-[theme(spacing.4/50)]:flex', 'max-[theme(--spacing-4/50)]:flex'],
 
+  // `theme(…)` calls in another CSS function is replaced correctly.
+  // Additionally we remove unnecessary whitespace.
+  ['grid-cols-[min(50%,theme(spacing.80))_auto]', 'grid-cols-[min(50%,var(--spacing-80))_auto]'],
```

Comment:
This doesn't seem to actually test the whitespace removal? Maybe something like this:
```suggestion
  ['grid-cols-[min(50%_,_theme(spacing.80))_auto]', 'grid-cols-[min(50%,var(--spacing-80))_auto]'],
```

Reply:
Copy & paste issue 🤦‍♂️